src/utils/appeal-utils.js

```diff
@@ -18,6 +18,11 @@ export function transformAppealDataAttributes(appeal) {
       dispute: {
         ...round.dispute,
         lastRoundId: parseInt(round.dispute.lastRoundId, 10),
+        rounds: round.dispute.rounds.map(round => ({
+          ...round,
+          number: parseInt(round.number, 10),
```

Comment:
maybe we can do those parse when we are transforming the dispute atrributes , `transformResponseDisputeAttributes` inside `dispute-utils`

Reply:
Not sure i understand what you mean,  the dispute for the appeals don't have the full data so don't think we should use`transformResponseDisputeAttributes`

Reply:
forgot about it , i read dispute-utils in the file name haha